docs/tables/config_ini.md

```diff
@@ -0,0 +1,121 @@
+# Table: config_ini
```

Comment:
Are there any other features/things worth calling out, like the env variable reference? Maybe describing and showing how we handle subsections, nested keys (and limitation of showing 1 level deep), and how multi-line values are handled?

Also, do we have any examples here showing some basic conversions, e.g., string->int, string->bool, that a user may want to do since `value` is always a string type?